client/ledsign/led_sign_client.js

```diff
@@ -67,4 +67,75 @@ function updateSignText(signData, signIp) {
   });
 }
 
+/**
+ * This function adds the text of a message to a queue of messages to be
+ * displayed on the LED sign.
+ * @param {Object} signMessage An object containing the desired message a user
+ * wants to send to a sign.
+ * @param {string} signIp - The IP address of the sign.
+ * @returns {Promise} Promise object which will contain the message from the
+ * sign and if an error occurred.
+ */
+function addMessageToQueue(signMessage, signIp) {
+  const client = new services.LedSignClient(
+    `${signIp}:50052`,
+    grpc.credentials.createInsecure()
+  );
+
+  const signMessageRequest = new messages.LedSignRecord();
+
+  signMessageRequest.setText(signMessage.text);
+  signMessageRequest.setBrightness(signMessage.brightness);
+  signMessageRequest.setScrollSpeed(signMessage.scrollSpeed);
+  signMessageRequest.setBackgroundColor(signMessage.backgroundColor);
+  signMessageRequest.setTextColor(signMessage.textColor);
+  signMessageRequest.setBorderColor(signMessage.borderColor);
+
+  signMessageRequest.setMessage(signMessage);
+
+  return new Promise(function(resolve, reject) {
+    client.addMessageToQueue(signMessageRequest, function(err, response) {
+      if (err) reject({ message: 'Message not added to queue', error: true });
+      resolve({ message: response, error: false });
+    });
+  });
+}
+
+/**
+ * This function clears the queue of messages to be displayed on the LED sign.
+ * @param {Object} signMessage An object containing the desired message a user
+ * wants to send to a sign.
+ * @param {string} signIp - The IP address of the sign.
+ * @returns {Promise} Promise object which will contain the message from the
+ * sign and if an error occurred.
+ */
+function clearMessageQueue(signMessage, signIp) {
+  const client = new services.LedSignClient(
+    `${signIp}:50052`,
+    grpc.credentials.createInsecure()
+  );
+
+  const clearMessageRequest = new messages.LedSignMessage();
+  clearMessageRequest.setMessage(signMessage);
+
+  return new Promise(function(resolve, reject) {
+    client.clearMessageQueue(clearMessageRequest, function(err, response) {
+      if (err) reject({ message: 'Message queue not cleared', error: true });
```

Comment:
`err` -> `error` xd

Reply:
actually would it break if we used error or would we be chilling bc error is used as a flag in the message back

Reply:
i just put it as err because the other methods that were already there used err hahah